WalletWasabi/Bases/ConfigBase.cs

```diff
@@ -21,12 +21,12 @@ protected ConfigBase(string filePath)
 	}
 
 	/// <inheritdoc />
-	public string FilePath { get; private set; } = null;
+	public string FilePath { get; private set; } = "";
 
 	/// <inheritdoc />
 	public void AssertFilePathSet()
 	{
-		if (FilePath is null)
+		if (string.IsNullOrWhiteSpace(FilePath))
```

Comment:
If we want to go with `= "";`, then the check is wrong. It should be `string.IsNullOrEmpty(FilePath)`.

Or simply use `= string.empty` for the property. Then the check can stay like this.

See [this](https://docs.microsoft.com/en-us/dotnet/api/system.string.isnullorwhitespace?view=net-6.0) and [this](https://docs.microsoft.com/en-us/dotnet/api/system.string.isnullorempty?view=net-6.0#system-string-isnullorempty(system-string)).

Reply:
It seems like we are deliberately using null as an indicator of FilePath was not set. Keep it nullable.